packages/js-sdk/tests/template/stacktrace.test.ts

```diff
@@ -249,3 +249,26 @@ buildTemplateTest('traces on addMcpServer', async () => {
     Template().fromBaseImage().addMcpServer('exa')
   }, 'addMcpServer')
 })
+
+buildTemplateTest(
+  'traces on devcontainerPrebuild',
+  async ({ buildTemplate }) => {
+    let template = Template().fromTemplate('devcontainer')
+    template = template.skipCache().devcontainerPrebuild(nonExistentPath)
+    await expectToThrowAndCheckTrace(async () => {
+      await buildTemplate(template)
+    }, 'devcontainerPrebuild')
+  }
+)
+
+buildTemplateTest(
+  'traces on setDevcontainerStart',
+  async ({ buildTemplate }) => {
+    const template = Template()
+      .fromTemplate('devcontainer')
+      .setDevcontainerStart(nonExistentPath)
```

Comment:
above you're using let template syntax but here a different one, can you settle on one style please?